_includes/faq.md

```diff
@@ -0,0 +1,23 @@
+## FAQ
+
+### Why is there no agenda?
+
+We call this "Everybody talks everybody listens"™️.
+Each participant is given a slot to present the subject of their choosing. 
+Depending on the attendance and session the slot may be between 5 and 30 minutes long.
+   
+### Should I hurry?
+
+That depends... But given the active participation format of REST Fest, availability
+is limited. We prefer a smaller crowd and encourage active knowledge sharing.
+   
+### Do I have to talk about RESTful APIs?
+
+Certainly not. It's perfectly fine to present REST-ish, REST-like and even GraphQL ;). 
+In fact, you can even talk about something else if you think it will be interesting to 
+other participants (see the question below). 
+
+### Do I have to present an API?
+
+Not necessarily. You can talk about what you are passionate about. Surely, API-related
+subjects are preferred but there have been exceptions.
```

Comment:
The last two questions are pretty similar. Can we combine them? Folks are certainly welcome to talk about 🐱's, etc. Our objective is to build better community within the REST and API space--hence, everyone talks; everyone listens. 😃